package/yast2-firewall.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Oct 22 20:44:20 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- do not warn user about closed ssh port if firewall is disabled
```

Comment:
NP: Well, the warning is not exactly about the closed port but about a possible not access :wink: but the fix looks good to me :yum: